bin/cij_plotter

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python
+"""
+ CIJOE Metric Plotter
+"""
+import argparse
+import hashlib
+import pprint
+import copy
+import sys
+import os
+import matplotlib.ticker as mticker
+import matplotlib.pyplot as plt
+import yaml
+import cij.analyser
+import cij.util
+import cij
+
+HATCHES = ['o', 'O', '.', '\\', '//', '-', '+', 'x', '\\', '*']
+
+MARKERS = ["+", "o", "s", "x", "h", "d", "*", "<", ">", "1", "*", "p"]
+
+METRICS = ["iops", "bwps", "lat"]
+
+METRIC_DEFAULT_UNIT = {
+    "iops": 'k',
+    "bwps": "MiB",
+    "lat": "usec"
+}
+
+KEY_DESCR = {
+    "iops": 'I/O operations Per Second',
+    "bwps": "BandWidth Per Second",
+    "lat": "Latency",
+
+    "bs": "Block-Size",
+    "iodepth": "I/O-Depth"
+}
+
+
+def collect_metrics(args):
+    """Collect all metrics found in 'metrics.yml' of testcase _aux folders"""
+
+    metrics = []
+    for root, _, _ in os.walk(args.output):
+        yml_fpath = os.path.join(root, "_aux", "metrics.yml")
+        if not (root.endswith(".sh") and os.path.exists(yml_fpath)):
+            continue
+
+        with open(yml_fpath, 'r') as yml_file:
+            metrics.extend(yaml.safe_load(yml_file))
+
+    return metrics
+
+
+def metrics_to_pdata(args, metrics):
+    """Transform the given metrics into plot-data"""
+
+    pdata = {}
+    for metric in metrics:                  # Transform metrics to plot-data
+        ctx = copy.deepcopy(metric["ctx"])
```

Comment:
We could also make this independent on `metrix["ctx"]` by doing `metric.get("ctx", {})`

Reply:
Oh, that is much better! Fixing...